.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish
+on: [push]
```

Comment:
Instead of removing `.bumpversion.cfg`, I strongly suggest making a change here: release when the GitHub release is published, instead of on push. That should fix the concerns about unexpected releases. See https://github.com/scrapy/itemadapter/blob/master/.github/workflows/publish.yml

That said, usually `bump2version` is the last thing you do, so even with `push` here I’m not that concern. It may be weird the first few times you use it, but in my experience you get used to it and it gives quite some piece on mind.

Reply:
Ok, I've restored it in 20b6365. I was having trouble, because the version was `1.2.0a1` and `bump2version` was rolling it to `1.2.1a1`, so I think I'll do a manual tag and release this time (instead of learning how to address this edge case), and then use `bump2version` going forward, as we won't have alpha tags.

I am unclear how the itemadapter workflow functions, since `published` is an event that relates to GitHub Packages, but that repository has no GitHub Packages. It does have Releases, but `scrapyd-client` does not, and personally, I prefer to just have tags in GitHub and leave the releases/packages to PyPI.

Reply:
Oh, nevermind, `published` is also an event for Releases. But, I prefer not to create those (and there aren't any yet), so I think we can leave it as `push` for now.

Reply:
> the version was 1.2.0a1 and bump2version was rolling it to 1.2.1a1

As a workaround, you could change the version to 1.2.0 in the bump2version configuration right before calling bump2version.